.travis.yml

```diff
@@ -23,4 +31,4 @@ after_success:
   - bash <(curl -s https://codecov.io/bash)
 cache:
   directories:
-    - $HOME/.pub-cache
+    - ＄HOME/.pub-cache
```

Comment:
there's something fishy about this `$` sign here 🤔

Reply:
not anymore. fixed